carts/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http import JsonResponse
+from django.views import View
+
+from carts.models import Cart
+from users.models import User
+from products.models import Product, Size
+from users.utils import login_deco
+
+class CartView(View):
+    @login_deco
```

Comment:
```suggestion
    @login_decorator
```
이거는 제가 놓친 부분인데, 데코레이터 이름도 같이 수정 부탁드리겠습니다.

Reply:
users.utils 파일 데코레이터 참조해서 login_deco 로 수정했습니다.